hal/mbed_ticker_api.c

```diff
@@ -41,8 +41,9 @@ static void initialize(const ticker_data_t *ticker)
 
     const ticker_info_t *info = ticker->interface->get_info();
     uint32_t frequency = info->frequency;
+
+    MBED_ASSERT(info->frequency == 0);
```

Comment:
Please review latest failures. Looks like we are hitting this assert in the tests. ERROR reported for all platforms there.

Reply:
https://github.com/ARMmbed/mbed-os/pull/11439#issuecomment-534051644